server/core/src/https/v1.rs

```diff
@@ -971,6 +971,44 @@ pub async fn account_delete_id_unix_credential(
     to_axum_response(res)
 }
 
+pub async fn identify_user(
+    State(_state): State<ServerState>,
+    Extension(_kopid): Extension<KOpId>,
+    Path(id): Path<String>,
+    Json(user_state): Json<IdentifyUserRequest>,
+) -> impl IntoResponse {
+    let _filter = filter!(f_id(id.as_str()));
+    trace!("{:?}", &user_state);
+    match user_state {
+        IdentifyUserRequest::Start => {
+            // here we need to check if the user can use the identity verification feature
```

Comment:
So here you'll need to make a new type of "event". You can see this in server/lib/src/idm/serviceaccount.rs where an GenerateApiTokenEvent is used. We want to wrap every "request" to an "event" type which completely encapsulates the request, this way we can always forge events in testing to simulate what was sent to us via the event layers. So have a look an trace something like that to undestand the layers here.

Reply:
Done!